src/main/java/org/sonarlint/intellij/actions/ReviewSecurityHotspotAction.kt

```diff
@@ -40,10 +40,13 @@ import org.sonarlint.intellij.core.BackendService
 import org.sonarlint.intellij.core.ProjectBindingManager
 import org.sonarlint.intellij.finding.hotspot.LiveSecurityHotspot
 import org.sonarlint.intellij.ui.review.ReviewSecurityHotspotDialog
+import org.sonarsource.sonarlint.core.clientapi.backend.hotspot.HotspotStatus
+import org.sonarsource.sonarlint.core.commons.HotspotReviewStatus
 
-class ReviewSecurityHotspotAction(private var serverFindingKey: String? = null) : AbstractSonarAction(
-    "Review Security Hotspot", "Review Security Hotspot Status on SonarQube", AllIcons.Actions.BuildLoadChanges
-), IntentionAction, PriorityAction, Iconable {
+class ReviewSecurityHotspotAction(private var serverFindingKey: String? = null, private var status: HotspotReviewStatus? = null) :
+    AbstractSonarAction(
+        "Review Security Hotspot", "Review Security Hotspot Status on SonarQube", AllIcons.Actions.BuildLoadChanges
```

Comment:
At some point we need to settle on an icon, or no icon if there is no good candidate